Chap_API_Publish.tex

```diff
@@ -195,25 +230,33 @@ \subsection{Range of Published Data}
 Undefined range.
 %
 \declareconstitemvalue{PMIX_RANGE_RM}{1}
-Data is intended for the host environment, or lookup is restricted to data published by the host environment.
+Published data and generated events are restricted to processes executing under the same instance of the host environment as the publisher or event creator.  Lookup of data is restricted to data published by processes running under the same instance of the host environment as the requester.
 %
 \declareconstitemvalue{PMIX_RANGE_LOCAL}{2}
```

Comment:
Reading this text I stumbled again on the question of wether ranges are "nested" or not. The new text in `PMIx_Lookup` below makes it very clear that it is not, but some forewarning here (or a ref to the lookup rules  `\ref{chap:pub:retrules}.`) could help clarify.

Reply:
Per IAWG on Aug. 15, 2022:
 * We should make clean that though the ranges are overlapping in definition, they are not nested. So ranges must match.
 * We should add the forward reference in the top paragraph of this section to the lookup section where this is more clearly noted.